spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/eventhub/DefaultEventHubClientFactory.java

```diff
@@ -80,6 +85,8 @@ public void initCheckpointConnectionString(String checkpointStorageAccount) {
 
     private EventHubClient createEventHubClient(String eventHubName) {
         try {
+            EventHubClientImpl.USER_AGENT = USER_AGENT + "/" + EventHubClientImpl.USER_AGENT;
```

Comment:
We only need to set this once, rather than for each client creation

Reply:
Yes, will move to constructor.